README.md

```diff
@@ -65,83 +65,68 @@ For more information and instructions, see the Kubernetes blog post, [Kubernetes
 
 ##### 4.2. For Fibre Channel, configure the relevant zoning from the storage to the host.
 
-
 <br/>
 <br/>
 <br/>
 
-
 ## Installation
 
-### Install the operator
+### Installing the driver with GitHub
 
+The operator for IBM® block storage CSI driver can be installed directly with GitHub. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
 
-#### 1. Download the manifest from GitHub.
+Use the following steps to install the operator and driver, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
 
-```bash
-curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/master/deploy/installer/generated/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
-```
+**Note:** Before you begin, you may need to create a user-defined namespace. Create the project namespace, using the `kubectl create ns <namespace>` command.
 
-#### 2. (Optional): If required, update the image fields in the ibm-block-csi-operator.yaml.
-**Note**: Updating the namespace to a user-defined namespace may be necessary in order to ensure consistency and avoid trouble installing the operator.
+#### 1.  Install the operator.
 
-#### 3. Create a namespace.
+##### 1. Download the manifest from GitHub.
```

Comment:
shouldn't this be [i. ...](https://github.com/IBM/ibm-block-csi-driver/blob/4d200fe1ccb6867cc4556cd206bda452bdcae65d/docs/csi_ug_install_operator_github.md)?

Reply:
its a question if we want (now) to start working with indentations ןמ איק README, we dont have it in the rest of the file

Reply:
I added subs (1.1., 1.2. , 2.1. etc.)

Reply:
I think we should use the "i" for sub-steps, to avoid numbering reference confusion

Reply:
why here and not in the rest of the file? we never used it

Reply:
because the installation and upgrade steps were changed, so multiple steps became one step (with sub-steps)

Reply:
I agree that for consistency, it should be aligned accordingly throughout the readme, as it is in the upcoming docs dir

Reply:
when the docs dir will be merged we could just link to them, I'm not seeing why we need to align to them now.

Reply:
because the current upgrading section (in this readme) refers to a step number in the installation section (in this readme), and it's important to perform all its sub-steps, which might be missed if they are numbered similarly

Reply:
they are numbered as sub steps.. (1.1, 1.2..) as they alwas have

Reply:
right, previously, the upgrade referred to numbered steps separately, but now, it refers to only 1 step, which has sub-steps, so the distinction between the steps and the sub-steps became more important